src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -0,0 +1,278 @@
+//# sourceMappingURL=/sm/bf59b3ee4485b26143b77d9336e0e09c1be06821ed3eec513e4a10039d8ca259.map
+
+/**
+ * Path to this script in the global tribe Object.
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets.commerce.gateway.stripe = tribe.tickets.commerce.gateway.stripe || {};
+
+/**
+ * This script Object for public usage of the methods.
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets.commerce.gateway.stripe.checkout = {};
+
+( ( $, obj, Stripe, ky ) => {
+	'use strict';
+	const $document = $( document );
+
+	/**
+	 * Pull the variables from the PHP backend.
+	 *
+	 * @since TBD
+	 *
+	 * @type {Object}
+	 */
+	obj.checkout = tecTicketsCommerceGatewayStripeCheckout;
+
+	/**
+	 * Checkout Selectors.
+	 *
+	 * @since TBD
+	 *
+	 * @type {Object}
+	 */
+	obj.selectors = {
+		cardElement: '#tec-tc-gateway-stripe-card-element',
+		cardErrors: '#tec-tc-gateway-stripe-card-errors',
+		submitButton: '#tec-tc-gateway-stripe-checkout-button',
+	};
+
+	/**
+	 * Stripe JS library.
+	 *
+	 * @since TBD
+	 *
+	 * @type {Object|null}
+	 */
+	obj.stripeLib = Stripe( obj.checkout.publishableKey );
+
+	/**
+	 * Stripe JS library elements.
+	 *
+	 * @since TBD
+	 *
+	 * @type {Object|null}
+	 */
+
+	/**
+	 * Settings for the Card Element from Stripe.
+	 *
+	 * @since TBD
+	 *
+	 * @return {{style: {base: {color: string}}}}
+	 */
+	obj.getOptions = () => {
+		return {
```

Comment:
Can we possibly receive this from the backend? Would be interesting to have it like that and have that array with the variables via hook so people can customize stuff.